cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/NamespaceCannotBeDeletedException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.namespace;
+
+import co.cask.cdap.common.exception.CannotBeDeletedException;
+
+/**
+ * {@link CannotBeDeletedException} thrown when a namespace cannot be deleted
```

Comment:
(nit) You are documenting `NamespaceCannotBeDeletedException` class though. It should be
`
// Thrown when a namespace cannot be deleted
`